tests/io/test_DataTextFileOperations.py

```diff
@@ -0,0 +1,13 @@
+import sys
```

Comment:
I am not sure what this file is doing.

Reply:
@MRVermeulenDeltares we should create unit tests with mock instead of data access tests

Reply:
https://issuetracker.deltares.nl/browse/DFAST-109